docs/pages/management/operations/scaling.mdx

```diff
@@ -42,6 +42,19 @@ teleport:
     max_users: 1000
 ```
 
+## Agent Configuration
+
+Consider bumping the default backoff parameters for agent caches. By default agents are fairly aggressive in trying
```

Comment:
Have we mentioned the agent cache in this page yet? If not, we should describe what an agent's cache does before this paragraph.